test/parallel/test-http2-client-request-options-errors.js

```diff
@@ -48,9 +67,9 @@ server.listen(0, common.mustCall(() => {
             [option]: types[type]
           }), {
             name: 'TypeError',
-            code: 'ERR_INVALID_ARG_VALUE',
-            message: `The property 'options.${option}' is invalid. ` +
-                    `Received ${inspect(types[type])}`
+            code: 'ERR_INVALID_ARG_TYPE',
+            message: `The "options.${option}" property must be of type ${optionsToTest[option]}. ` +
+                    `Received ${determineSpecificType(types[type])}`
```

Comment:
We should avoid checking for the error message as much as possible, checking for the error code is enough.

Reply:
understood removing this

Reply:
Done